docs: remove links to Registry

mrshll1001 opened the pull request:
See https://opendataservices.plan.io/issues/55230

Removed links to data.threesixtygiving.org and replaced with links to the Quality Dashboard.

mrshll1001 committed:
docs: remove links to Registry

See https://opendataservices.plan.io/issues/55230

Removed links to data.threesixtygiving.org and replaced with links to
the Quality Dashboard.

mariongalley reviewed (changes requested):
Please can you change Quality Dashboard to Data Quality Dashboard?

mrshll1001 committed:
docs: update strings to "Data Quality Dashboard"

Erroneously had stated "360Giving Quality Dashboard". This commit fixes
that

KDuerden reviewed (approved):
I have reviewed these changes which update the name and where appropriate URL for the Data Registry and Data Quality Checker. These are mostly in non-normative parts of the documentation site, however some updates make changes to the normative, Standard documentation.

These changes do not change the meaning of the text and updates to links point to the  current relevant tool, I believe we can approve this change without the need for a PATCH.

mrshll1001 committed:
docs: replace instances of Data Quality Tool

As per https://opendataservices.plan.io/issues/55230, replaced instanced
of "Data Quality Tool" with "Data Quality Checker".

Fixed merge conflicts present after merging #428, which removed
non-normative guidance.

Have not yet replaced instances of "DQT" in docs

mrshll1001 committed:
docs: updated footer to replace Quality Dashboard

There was a stray "Quality Dashboard" in footer.json, which has now been
replaced by "Data Quality Dashboard"

mrshll1001 committed:
Merge branch 'main' into mm-remove-links-to-registry
